docs/alexa-apis.rst

```diff
@@ -330,103 +324,101 @@ Alexa customers have access to two default lists: Alexa to-do and Alexa shopping
-  :returns Object: A JSON object with the specific list info.
+    :param name: List name.
+    :param active: list status, defaults to active (only value accepted for now)
+    :returns Object: A JSON object with the specific list info.
 
-.. js:function:: Lists.updateList(listId, name, state = 'active', version)
+  .. js:method:: updateList(listId, name, state = 'active', version)
 
-  Updates list with the name, state, and version.
+    Updates list with the name, state, and version.
 
-  :param listId: List ID.
-  :param state: list status, defaults to active (only value accepted for now)
-  :param version: List version.
-  :returns Object: A JSON object with the specific list info.
+    :param listId: List ID.
+    :param state: list status, defaults to active (only value accepted for now)
+    :param version: List version.
+    :returns Object: A JSON object with the specific list info.
 
-.. js:function:: Lists.deleteList(listId)
+  .. js:method:: deleteList(listId)
 
-  Deletes a list by ID.
+    Deletes a list by ID.
 
-  :param listId: List ID.
-  :returns: undefined. HTTP response with 200 or error if any.
+    :param listId: List ID.
+    :returns: undefined. HTTP response with 200 or error if any.
 
-.. js:function:: Lists.getListItem(listId, itemId)
+  .. js:method:: getListItem(listId, itemId)
 
-  Creates a new list with the name and state.
+    Creates a new list with the name and state.
 
-  :param listId: List ID.
-  :param itemId: Item ID.
-  :returns Object: A JSON object with the specific list info.
+    :param listId: List ID.
+    :param itemId: Item ID.
+    :returns Object: A JSON object with the specific list info.
 
-.. js:function:: Lists.createItem(listId, value, status = 'active')
+  .. js:method:: createItem(listId, value, status = 'active')
 
-  Creates a new list with the name and state.
+    Creates a new list with the name and state.
 
-  :param listId: List ID.
-  :param value: Item name.
-  :param status: item status, defaults to active. Other values accepted: 'completed'
-  :returns Object: A JSON object with the specific item info.
+    :param listId: List ID.
+    :param value: Item name.
+    :param status: item status, defaults to active. Other values accepted: 'completed'
+    :returns Object: A JSON object with the specific item info.
 
-.. js:function:: Lists.updateItem(listId, itemId, value, status, version)
+  .. js:method:: updateItem(listId, itemId, value, status, version)
 
-  Creates a new list with the name and state.
+    Creates a new list with the name and state.
 
-  :param listId: List ID.
-  :param itemId: Item ID.
-  :param value: Item name.
-  :param status: item status. Values accepted: 'active | completed'
-  :returns Object: A JSON object with the specific item info.
+    :param listId: List ID.
+    :param itemId: Item ID.
+    :param value: Item name.
+    :param status: item status. Values accepted: 'active | completed'
```

Comment:
Uppercase `Item`

Reply:
@omenocal btw, if only those 2 values are accepted, please create a type with those 2 values instead of just saying `string`